packages/web/src/components/createCell.tsx

```diff
@@ -107,7 +107,7 @@ export type CellSuccessProps<
     Omit<QueryOperationResult<TData, TVariables>, 'loading' | 'error' | 'data'>
   >
   updating?: boolean
-} & A.Compute<CellSuccessData<TData>> // pre-computing makes the types more readable on hover
+} & A.Compute<Partial<CellSuccessData<TData>>> // pre-computing makes the types more readable on hover
```

Comment:
I think this would do the trick, but I wonder if it would make more sense to instead remove `Guaranteed` from `type CellSuccessData` further up.  (`Guaranteed` itself seems obsolete, along with the comments related to the old behavior.)

I think that would look like changing this:
```ts
export type CellSuccessData<TData = any> = Omit<Guaranteed<TData>, '__typename'>
```
to this:
```ts
export type CellSuccessData<TData = any> = Omit<TData, '__typename'>
```

The end result should be that the type of `TData` is unmodified.  With normal usage it would just be the generated query type as-is (though still with `__typename` omitted).  This seems more straightforward than making the type `Guaranteed` only to make it `Partial` again -- unless there's a good reason to keep the _exported_ `CellSuccessData` type `Guaranteed`.